lib/gems/pending/appliance_console/key_configuration.rb

```diff
@@ -63,6 +66,17 @@ def activate
       end
     end
 
+    def save_temp_key
+      FileUtils.mv(KEY_FILE_TEMP, KEY_FILE)
+    rescue StandardError => e
```

Comment:
No need to specify `StardardError` here `rescue => e` will work fine (like [here](https://github.com/ManageIQ/manageiq-gems-pending/pull/211/files#diff-f53388092c798ee64087388537f48064R98))